test/test_collect.jl

```diff
@@ -1,44 +1,44 @@
 using IndexedTables: collect_columns_flattened
 
 @testset "collectnamedtuples" begin
-    v = [@NT(a = 1, b = 2), @NT(a = 1, b = 3)]
-    @test collect_columns(v) == Columns(@NT(a = Int[1, 1], b = Int[2, 3]))
+    v = [(a = 1, b = 2), (a = 1, b = 3)]
+    @test collect_columns(v) == Columns((a = Int[1, 1], b = Int[2, 3]))
```

Comment:
It'd be nice to have a constructor for `Columns` using keyword args, so a level of parens can be dropped.

```julia
@test collect_columns(v) == Columns(a = Int[1, 1], b = Int[2, 3])
```

Reply:
I think there is such a constructor; maybe this is just checking that passing a (named)tuple also works.